cypress/integration/admin_portal/pages/enterprise_code_management.js

```diff
@@ -4,30 +4,85 @@ class CodeManagementPage {
   }
 
   getLabels(labelName) {
-    const labels = {Email: 1, Company: 2, Codes:3}
+    const labels = { Email: 1, Company: 2, Codes: 3 }
     return cy.get(`.form-group:nth-of-type(${labels[labelName]}) label`)
   }
 
-  getRequestCodesButton(){
+  getRequestCodesButton() {
     return cy.get('.btn.btn-primary')
   }
 
-  getCancelButton(){
+  getCancelButton() {
     return cy.get('.btn.btn-link.ml-3.form-cancel-btn')
   }
 
-  getInvalidFeedback(){
+  getInvalidFeedback() {
     return cy.get('[name="emailAddress"]~.invalid-feedback')
   }
 
-  getSuccessMessage(){
-      return cy.get('.alert-dialog .message')
+  getSuccessMessage() {
+    return cy.get('.alert-dialog .message')
   }
 
-  requestMoreCodes(){
+  requestMoreCodes() {
     cy.get('.request-codes-btn.btn.btn-primary').click()
   }
 
+  getCoupon() {
+    return cy.get('.coupon')
+  }
+
+  getCouponDetailsRow() {
+    return cy.get('.expanded .metadata.row')
+  }
+
+  getCouponDetailsRowText() {
+    return cy.get('small.text-light')
```

Comment:
if you just want the text you can return it from here as well

Reply:
This was an extra, so I have removed it.